### ISSUE TITLE
Like users from file error

### ISSUE THREAD

**Issue by penarubah** [ohld/igbot#60]
try to run this code by modifying follow_users_from_file.py
But it give long error.

run this script by: `python penarubah_like_3_media_from_file.py user_to_follow.txt`

the code is:
``` python
"""
    instabot example

    Workflow:
        Take users from input file and like 3 latest media of them.
        The file should contain one username per line!
"""

import sys
import os

sys.path.append(os.path.join(sys.path[0],'../'))
from instabot import Bot

users_file = "users_to_follow.txt"

bot = Bot()
users_to_like = bot.read_list_from_file(users_file)
if not users_to_like:
    exit()
bot.login()
bot.like_users(users_to_like, nlikes=3)
``` 

![screenshot_5](https://cloud.githubusercontent.com/assets/25403367/23161638/81af0270-f85d-11e6-815a-b506f1463525.png)
![screenshot_4](https://cloud.githubusercontent.com/assets/25403367/23161637/81aa608a-f85d-11e6-9a96-c82d6d69f0c0.png)
[users_to_follow.txt](https://github.com/ohld/instabot/files/789867/users_to_follow.txt)

**Comment by maxkrivich** [comment]
@penarubah what encoding do you use?

**Comment by ohld** [comment]
Fixed in 0.3.2.2. Reopen of necessary.
